test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -611,12 +611,12 @@ Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows"
         $realDir = Join-Path $TestPath "subdir"
         $realDir2 = Join-Path $TestPath "second-subdir"
         $nonDir = Join-Path $TestPath "not-a-dir"
-        $hardLinkToFile = Join-Path $TestPath "hard-to-file.txt"
-        $symLinkToFile = Join-Path $TestPath "sym-link-to-file.txt"
-        $symLinkToDir = Join-Path $TestPath "sym-link-to-dir"
-        $symLinkToNothing = Join-Path $TestPath "sym-link-to-nowhere"
-        $dirSymLinkToDir = Join-Path $TestPath "symd-link-to-dir"
-        $junctionToDir = Join-Path $TestPath "junction-to-dir"
+        $hardLinkToFile = Join-Path $TestPath "[hard-to-file].txt"
```

Comment:
Since we haven't explicit tests please add new comment before the line that we intentionally use wildcard chars in follow names for testing.